includes/Core/Util/Debug_Data.php

```diff
@@ -347,4 +349,26 @@ function ( Module_With_Debug_Fields $module ) {
 
 		return array_merge( array(), ...$fields_by_module );
 	}
+
+
+	/**
+	 * Gets capabilities for the current user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_capabilities_field() {
```

Comment:
A bit nitpicky but please move the definition of this method to go after `get_required_scopes_field` and before `get_module_fields` for consistency with the others.